### PR TITLE
FLINK-35402 [flink-operator][Deployment] add labels to metadata

### PR DESCRIPTION
**Pull request by luismacosta**
FLINK-35402 [flink-operator][Deployment] add labels to metadata - https://issues.apache.org/jira/browse/FLINK-35402

### CODE REVIEW

File: helm/flink-kubernetes-operator/templates/flink-operator.yaml [hunk 1]
```diff
@@ -23,6 +23,11 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "flink-operator.labels" . | nindent 4 }}
```
[the remark this answers is not in the document]
Comment (gyfora) [reply helm/flink-kubernetes-operator/templates/flink-operator.yaml:25]: Is there any reason why you could not add the other labels to `flink-operator.labels` as well?

[the remark this answers is not in the document]
Comment (luismacosta) [reply helm/flink-kubernetes-operator/templates/flink-operator.yaml:25]: From what I can see in [flink-operator.labels](https://github.com/apache/flink-kubernetes-operator/blob/main/helm/flink-kubernetes-operator/templates/_helpers.tpl#L54) there is no way to add more labels. 
To achieve that, my idea here was to add to metadata.labels the same the is configure at [spec.template.metadata.labels](https://github.com/apache/flink-kubernetes-operator/blob/main/helm/flink-kubernetes-operator/templates/flink-operator.yaml#L36-L41)

[the remark this answers is not in the document]
Comment (gyfora) [reply helm/flink-kubernetes-operator/templates/flink-operator.yaml:25]: Ok in that case you need to update the respective doc pages too please, as they would be incorrect:
image

[the remark this answers is not in the document]
Comment (luismacosta) [reply helm/flink-kubernetes-operator/templates/flink-operator.yaml:25]: docs updated

File: docs/content.zh/docs/operations/helm.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ The configurable parameters of the Helm chart and which default values as detail
...
-| operatorPod.labels                             | Custom labels to be added to the operator pod (but not the deployment).                                                                                        |                                                                                                                                                                                                                                                                                                |
+| operatorPod.labels                             | Custom labels to be added to the operator pod                                                                                        |                                                                                                                                                                                                                                                                                                |
```
Comment (gyfora) [inline docs/content.zh/docs/operations/helm.md:79]: the updated descriptions are not correct...
It should say ` Custom labels to be added to the operator pod  and deployment`

Comment (luismacosta) [reply docs/content.zh/docs/operations/helm.md]: ok. thanks.

**Pull request merged**